### PR TITLE
feat: new architecture with DevWorkspace operator (part 1 - server side)

### PR DESCRIPTION
**Pull request by l0rd**
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18887

image

image

image

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [x] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/administration-guide/partials/con_che-architecture-overview-with-dw.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+.High-level {prod-short} architecture
+image::administration-guide:architecture/{project-context}-high-level.png[]
+
+When {prod-short} is installed on a {platforms-name} cluster, the workspace controller is the only component that is deployed. A {prod-short} workspace is created immediately after a user requests it.
+
+.Additional resources
+
+* xref:administration-guide:che-workspace-controller-with-dw.adoc[]
+* xref:administration-guide:che-workspaces-architecture-with-dw.adoc[]
```
Comment (themr0c) [inline modules/administration-guide/partials/con_che-architecture-overview-with-dw.adoc]: These pages don't exist. (See link checker errors).

**Comment by l0rd** [comment]
@themr0c @MichalMaler it's in draft mode, it's not ready for review yet.

File: modules/administration-guide/partials/con_che-architecture-overview-with-dw.adoc (hunk 1 above)
Comment (MichalMaler) [reply modules/administration-guide/partials/con_che-architecture-overview-with-dw.adoc]: Was this caused by a fork from the old master branch?

**Comment by l0rd** [comment]
@MichalMaler this PR is ready for review. The second part (workspaces side changes) will be submitted in a separate PR after this one gets merged.

**Comment by l0rd** [comment]
@themr0c @MichalMaler I have solved the conflicts.

**Comment by themr0c** [comment]
Replaced by https://github.com/eclipse/che-docs/pull/2125

**Pull request closed without merging**